tests/deployment/test_client_server.py

```diff
@@ -89,18 +96,31 @@ def test_client_server_sklearn(
         # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4014
         pytest.skip("Skipping KNN, because FHE predictions and clear ones are differents.")
 
+    if (
+        get_model_name(model_class) == "SGDClassifier"
+        and parameters.get("n_classes") != 2
+        and getattr(model_class, "keywords", {}).get("fit_encrypted") is True
+    ):
+        # Skipping SGDClassifier for this test
+        # FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/4182
+        pytest.skip("Skipping SGDClassifier, because it only supports 2 classes.")
```

Comment:
why do you need to add this here ?

Reply:
Because we test sklearn models with more than 2 classes but SGDClassifier isn't compatible yet. Or am I wrong?

Reply:
yes but why now ? I mean why wasn't it already here, did you change somthing with SGDClassifier itself ?

Reply:
We were not testing it with `fit_encrypted = True` this skip is only when n_classes != 2 and `fit_encrypted = True`

Reply:
gotcha thanks !